coremltools/converters/mil/frontend/torch/internal_graph.py

```diff
@@ -6,7 +6,9 @@
 from collections import OrderedDict
 
 import torch
-from torch.fx.node import Node
+import torch.fx
+import torch.fx.immutable_collections
+import torch.export
```

Comment:
why do we need these three import?

Reply:
My vs code UI needs to import them in order to show type hints.

Reply:
Okay!